datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -164,4 +190,20 @@ private FlowIdentifier notify(String selector, SdxEvent acceptable) {
         }
 
     }
+
+    private String getBackupLocation(String datalakeName) {
+        String backupLocation = null;
+        DetailedEnvironmentResponse detailedEnvironmentResponse = environmentClientService.getByName(datalakeName);
```

Comment:
I see the datalakename here. 2 issues here:
- should be env
- should be crn between services